config/routes.rb

```diff
@@ -1,6 +1,10 @@
 Rails.application.routes.draw do
+  get '/users/:user_id/posts', to: 'posts#index'
+  get '/users/:user_id/posts/:id', to: 'posts#show'
+
+  resources :users, only: %i[index show]
```

Comment:
-   [ ] A better, more Rails way to configure these routes could be:
  
  ```ruby
  resources :users, only: [:index, :show] do
    resources :posts, only: [:index, :show]
  end
  ```

Reply:
@Lameck1  Thank you fo the suggestion, yes your are right, better this way 👍